datanode/entities/pagination.go

```diff
@@ -19,6 +19,8 @@ import (
 	"github.com/pkg/errors"
 )
 
+var defaultPageSize int32 = 1000
```

Comment:
should we make smaller pages? This seems massive?

Reply:
I dunno I thought it seemed rather small.. i mean ball park let's say it's 512 bytes per record on average (and I would think that's a significant overestimate) that's just a 500kb payload. The smaller you make it the more requests someone has to do to get the whole set if they want it, and there is definitely much more cost to someone doing 2 requests or 20. Really you want to minimise the number of requests/database queries; I doubt it makes a lot of difference to Postgres if you're getting 100 records or 10,000 but it's when there are millions it starts getting a bit more unmanageable

Reply:
Hum fair enough